sw/airborne/modules/sonar/sonar_bebop.c

```diff
@@ -36,22 +44,63 @@
 #include "state.h"
 #endif
 
+enum peak_search_state {INIT, BROADCASTING, SEARCHING_MIN, SEARCHING_PEAK};
+static enum peak_search_state search_state;
+
+/* conversion form index difference to distance based on time of flight
```

Comment:
since you already put comments before the defines (:+1:), please use doxygen comments so it will show up in the generated docs...